### PR TITLE
Port import automation executor to GKE

### PR DESCRIPTION
**Pull request by Fructokinase**
This PR contains files necessary to deploy the import automation executor as a GKE service with 20G memory and 10G disk in datcom-website-dev.

### CODE REVIEW

File: import-automation/executor/gke/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Deploy to GKE
+
+Import-automation executor can also be deployed to gke. After committing new changes, please do the following to deploy.
+
+1. cd to executor dir.
```
Comment (juliawu) [inline import-automation/executor/gke/README.md:5]: Suggestion: let's point to at least the relative path here?

File: import-automation/executor/gke/cloudbuild.push_image.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    name: gcr.io/cloud-builders/docker
+    entrypoint: "bash"
+    args:
+      - -c
+      - |
+        set -e
+        DOCKER_BUILDKIT=1 docker build -f Dockerfile --tag gcr.io/$_PROJECT_ID/datacommons-import-automation-executor:${_TAG} .
+        docker push gcr.io/$_PROJECT_ID/datacommons-import-automation-executor:${_TAG}
+
+timeout: 3600s
```
Comment (juliawu) [inline import-automation/executor/gke/cloudbuild.push_image.yaml:25]: This is a long timeout. Should it be shorter?

**Review by juliawu — APPROVED** [review]
Thanks for the changes! Just some minor comments.

File: import-automation/executor/gke/cloudbuild.push_image.yaml (hunk 2 above)
Comment (Fructokinase) [reply import-automation/executor/gke/cloudbuild.push_image.yaml:25]: Changed it to 10min, thanks!

File: import-automation/executor/gke/README.md (hunk 1 above)
Comment (Fructokinase) [reply import-automation/executor/gke/README.md:5]: Done.

**Pull request merged**